git-send-email.perl

```diff
@@ -920,7 +935,7 @@ sub validate_address {
 			valid_re => qr/^(?:quit|q|drop|d|edit|e)/i,
 			default => 'q');
 		if (/^d/i) {
-			return undef;
+			return;
```

Comment:
This is incorrect. The `validate_address` function clearly is expected to return an address.